python/GafferUI/PlugLayout.py

```diff
@@ -710,6 +755,9 @@ def update( self, section ) :
 				"<small>" + "&nbsp;( " + subsection.summary + " )</small>" if subsection.summary else ""
 			)
 
+			collapsible._qtWidget().setProperty( "gafferValueChanged", GafferUI._Variant.toVariant( subsection.valuesChanged ) )
+			collapsible._repolish()
```

Comment:
Elsewhere where we've done a repolish, it has been important for performance that we only do it when the property has actually changed. Can we guard this in the same way?

Reply:
Hows [this](https://github.com/GafferHQ/gaffer/pull/3895/commits/a75fed06f4afa9ed2fef8dc4f156d2767b0016e8)?